### ISSUE TITLE
SQLite - string parameters are not escaped

### ISSUE THREAD

**Issue by p-mitana** [huntlabs/database#30]
I tried to append a record with a string containing apostrophe.

Code:

```d
auto statement = CascataskDatabase.prepare(`UPDATE tasks SET title = :title WHERE task_id = :taskID ;`);
statement.setParameter("title", title);
statement.setParameter("taskID", taskID.to!string);
statement.execute();
```

Parameters: test = "Test'test", taskID is any UUID.

Result:

    hunt.database.DatabaseException.DatabaseException@../../../.dub/packages/hunt-database-1.1.3/hunt-database/source/hunt/database/driver/sqlite/Connection.d(122): SQL : UPDATE tasks
			SET title = 'Test'test' WHERE task_id = '*****' ;

**Comment by gaoxincheng** [comment]
Please pull the code again.

**Comment by zoujiaqing** [comment]
@the-Mitu this bug fixed, but waiting for release next vertion 1.2.0
